tf2_ros/src/create_timer_ros.cpp

```diff
@@ -36,15 +36,33 @@
 #include <functional>
 #include <mutex>
 #include <stdexcept>
+#include <memory>
 
 namespace tf2_ros
 {
 
 CreateTimerROS::CreateTimerROS(
   rclcpp::node_interfaces::NodeBaseInterface::SharedPtr node_base,
-  rclcpp::node_interfaces::NodeTimersInterface::SharedPtr node_timers)
-: node_base_(node_base), node_timers_(node_timers), next_timer_handle_index_(0)
+  rclcpp::node_interfaces::NodeTimersInterface::SharedPtr node_timers,
+  bool spin_thread)
```

Comment:
`spin_thread` doesn't seem quite right here. Since this has to do specifically with the timer processing, I think a different name here would be more appropriate so its more clear to users what this actually is for (e.g. allows the timer interface to be processed independently from the `base_node` application processing).

Reply:
could you give me some suggestions for naming ? how about `is_independent` ？

Reply:
I'm not really sure actually. I'd be interested what the maintainers here think (or if they feel its even worth parameterizing or should just always be enabled)